src/hooks/constants.ts

```diff
@@ -0,0 +1,244 @@
+import BigNumber from '@lib/bigNumber'
```

Comment:
a small nitpick on the filename. it should be `types.ts` since this is nothing but a types declaration file.

Reply:
We can do that after, its not related.